website/project/model.py

```diff
@@ -2359,15 +2371,19 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             user=user,
             foreign_user=foreign_user,
             params=params,
+            node=self,
+            original_node=params['node']
         )
 
         if log_date:
             log.date = log_date
+        log.save()
 
-        self.date_modified = log.date.replace(tzinfo=None)
+        if len(self.logs) == 1:
```

Comment:
Without digging too much into what is happening `datetime` discrepancies are due to mongo having slightly less precision than python does. When a model if first created it will have the milliseconds attached to the `datetime` but after being saved and loaded from mongo the milliseconds have been dropped. ODM should probably trim datetime at the python level just for consistency. I know we have a decent amount of tests that look weird due to this case.